build-configs/BuildConfigType.js

```diff
@@ -3,61 +3,70 @@
 import type { ThemeType } from './ThemeType'
 import type { TranslationsType } from 'translations'
 
+/**
+ * Build Configs
+ * These are the flow types of our build configs and therefore define the structure and available options.
```

Comment:
I think we should explain here the difference between build options and feature flags